client/packages/common/src/authentication/AuthContext.tsx

```diff
@@ -3,7 +3,7 @@ import { IntlUtils } from '@common/intl';
 import { AppRoute } from '@openmsupply-client/config';
 import { LocalStorage, useLocalStorage } from '../localStorage';
 import Cookies from 'js-cookie';
-import { addMinutes } from 'date-fns';
```

Comment:
Oh no, I though a tree shaker is smart enough :(

Reply:
there were some comments around the `date-fns` package, the way that the modules were exported can cause problems for webpack to implement tree-shaking. It did make a big difference for `lodash`.. so something isn't quite right.

Reply:
ok. retested and the `date-fns` is able to shake it like a pro.